shopinvader_schema_sale/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2023 Akretion (https://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Shopinvader Schema Sale",
+    "summary": "Add schema sale",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "website": "https://github.com/shopinvader/odoo-shopinvader",
+    "author": " Akretion",
+    "license": "AGPL-3",
+    "depends": [
+        "shopinvader_schema_address",
```

Comment:
Missing dependencies: `sale_cart` (for`typology`) , `sale_discount_display_amount` (for `discount_total` ...)